Assets/MixedRealityToolkit/InputModule/Scripts/Utilities/Managers/InputManager.cs

```diff
@@ -10,7 +16,7 @@
 using UnityEngine.Windows.Speech;
 #endif
 
-namespace HoloToolkit.Unity.InputModule
+namespace MixedRealityToolkit.InputModule.Utilities.Managers
```

Comment:
Don't like the placement of the InputManager, suggest it should be moved to the base of the MRTK.InputModule as it's a core class (as with other managers listed in this folder)

Reply:
I agree. Putting it at the base of the namespace would be ideal, since it's really the basis for the entire namespace existing.

Reply:
Agreed

Reply:
Ok, moved 
* InputManager
* MixedRealityCameraManager
* MixedRealityTeleport

To the root of the InputModule namespace